### PR TITLE
Updated the faculty list for Indiana University Bloomington.

### PR DESCRIPTION
**Pull request by sameer-patil**
Updating faculty for Indiana University Bloomington:
- Removed duplicate (David Crandall) for David J. Crandall (DBLP has only one entry)
- Geoffrey C. Fox (moved to UVA)
- Hamid R. Ekbia (moved to Syracuse)
- Steven Myers (moved to Apple Inc.; does not have Google Scholar; fixed the wrong Google Scholar pointer to some other Steven Myers in Florida)
- Süleyman Cenk Sahinalp (moved to NIH; has an adjunct appointment at UMD but it's not clear if that meets the criteria of being able to chair CS committees at UMD)
- Shuang Wang 0002 (moved to industry)
- Yuan Zhou 0007 (moved to UIUC; fixed wrong Google Scholar pointer)

### CODE REVIEW

File: csrankings-d.csv [hunk 1]
```diff
@@ -526,7 +526,6 @@ David Clark 0001,University College London,http://www0.cs.ucl.ac.uk/people/D.Cla
 David Cordes,The University of Alabama,http://cordes.cs.ua.edu,NOSCHOLARPAGE
 David Corne,Heriot-Watt University,https://www.hw.ac.uk/staff/uk/macs/david-wolfe-corne.htm,6jNgTK4AAAAJ
 David Coyle,University College Dublin,https://people.ucd.ie/d.coyle,9d9bOOUAAAAJ
-David Crandall,Indiana University,http://www.cs.indiana.edu/~djcran,8bQRH5YAAAAJ
```
Comment (emeryberger) [inline csrankings-d.csv]: DBLP lists David Crandall 0001 as an alias for David J. Crandall. Please fix. https://dblp.org/pid/c/DavidCrandall.html

File: csrankings-h.csv [hunk 2]
```diff
@@ -149,7 +149,7 @@ Hamid Bagheri,University of Nebraska,http://cse.unl.edu/~hbagheri,h_AYrscAAAAJ
 Hamid Beigy,Sharif University of Technology,http://sharif.edu/~beigy,0NiKG0EAAAAJ
 Hamid Dehghani,University of Birmingham,https://www.birmingham.ac.uk/staff/profiles/computer-science/academic-staff/dehghani-hamid.aspx,SIYS-lcAAAAJ
 Hamid R. Arabnia,University of Georgia,http://cobweb.cs.uga.edu/~hra,g86VLOUAAAAJ
-Hamid R. Ekbia,Indiana University,https://www.soic.indiana.edu/all-people/profile.html?profile_id=72,WPGTvh8AAAAJ
+Hamid R. Ekbia,Syracuse University,https://www.maxwell.syr.edu/directory/hamid-ekbia,WPGTvh8AAAAJ
```
Comment (emeryberger) [inline csrankings-h.csv]: Looks to no longer be in CS; please remove.

File: csrankings-s.csv [hunk 3]
```diff
@@ -1082,7 +1082,7 @@ Shuang Hao,University of Texas at Dallas,http://utdallas.edu/~shao,FgpHWKYAAAAJ
 Shuang Li 0002,CUHK (SZ),https://shuangli01.github.io,HxCZsCUAAAAJ
 Shuang Luan,University of New Mexico,https://www.cs.unm.edu/~sluan,JF8EmRMAAAAJ
 Shuang Wang 0001,Xidian University,https://web.xidian.edu.cn/shwang/,CeLvrv8AAAAJ
-Shuang Wang 0002,Indiana University,http://www.shuangwang.org,GnDGqKQAAAAJ
+Shuang Wang 0002,NVXClouds,http://www.shuangwang.org,GnDGqKQAAAAJ
```
Comment (emeryberger) [inline csrankings-s.csv]: Moves to industry relocate entries to `old/industry.csv`.

File: csrankings-s.csv [hunk 4]
```diff
@@ -1901,7 +1901,7 @@ Steven M. Nowick,Columbia University,http://www.cs.columbia.edu/~nowick,gaXiAekA
 Steven M. Seitz,University of Washington,http://cs.washington.edu/homes/seitz,-6cOYcYAAAAJ
 Steven Mills,University of Otago,http://www.cs.otago.ac.nz/staff/Steven_Mills,lfySPjwAAAAJ
 Steven Minsker,University of Arkansas - Little Rock,https://ualr.edu/computerscience/about/faculty-and-staff/steven-minsker,NOSCHOLARPAGE
-Steven Myers,Indiana University,http://wphomes.soic.indiana.edu/samyers,SQnCsswAAAAJ
+Steven Myers,Apple,https://www.linkedin.com/in/steven-myers-4351a12,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-s.csv]: See above re: moves to industry.

File: csrankings-s.csv [hunk 5]
```diff
@@ -2045,7 +2045,7 @@ Sule Gündüz,Istanbul Technical University,http://web.itu.edu.tr/sgunduz,E5OlkJ
 Sule Gündüz Ögüdücü,Istanbul Technical University,http://web.itu.edu.tr/sgunduz,E5OlkJMAAAAJ
 Sule Ögüdücü,Istanbul Technical University,http://web.itu.edu.tr/sgunduz,E5OlkJMAAAAJ
 Suleman Shahid,LUMS,https://lums.edu.pk/lums_employee/4407,NOSCHOLARPAGE
-Süleyman Cenk Sahinalp,Indiana University,https://www.soic.indiana.edu/all-people/profile.html?profile_id=291,NOSCHOLARPAGE
+Süleyman Cenk Sahinalp,National Institutes of Health,https://ccr.cancer.gov/staff-directory/s-cenk-sahinalp,O8wbTncAAAAJ
```
Comment (emeryberger) [inline csrankings-s.csv]: Put in `old/other.csv`.

File: csrankings-y.csv [hunk 6]
```diff
@@ -671,7 +671,7 @@ Yuan Yuan 0001,NWPU,https://teacher.nwpu.edu.cn/m/en/2018010291.html,NOSCHOLARPA
 Yuan Zhang 0004,Nanjing University,http://cosec.nju.edu.cn/#index.php/members/yuan-zhang,ZBV1qlMAAAAJ
 Yuan Zhang 0006,UESTC,https://www.en.scse.uestc.edu.cn/info/1085/2006.htm,7rWSrzsAAAAJ
 Yuan Zhang 0009,Fudan University,https://yuanxzhang.github.io,ZEgzyZcAAAAJ
-Yuan Zhou 0007,Indiana University,https://www.soic.indiana.edu/all-people/profile.html?profile_id=442,aR34e1gAAAAJ
+Yuan Zhou 0007,University of Illinois at Urbana-Champaign,https://yuanz.web.illinois.edu,j4Fshz0AAAAJ
```
Comment (emeryberger) [inline csrankings-y.csv]: Institution name should be "Univ. of Illinois at Urbana-Champaign".

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-d.csv (hunk 1 above)
Comment (sameer-patil) [reply csrankings-d.csv]: So this entry should be kept as is?

**Comment by sameer-patil** [comment]
Made the changes as requested.

File: csrankings-s.csv (hunk 4 above)
Comment (emeryberger) [reply csrankings-s.csv]: This entry needs to be removed from `csrankings-*.csv`.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-s.csv (hunk 4 above)
Comment (sameer-patil) [reply csrankings-s.csv]: done

File: csrankings-d.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings-d.csv]: You need both entries.

Comment (sameer-patil) [reply csrankings-d.csv]: Yes; the latest commit has both entries.

**Pull request merged**